packages/lesswrong/components/hooks/useAnalytics.ts

```diff
@@ -186,6 +196,7 @@ export const useAnalyticsSeries = (props: UseAnalyticsSeriesProps): {
   error?: ApolloError
 } => {
   const { userId, postIds, startDate: propsStartDate, endDate: propsEndDate } = props;
+  const cachedFirst = !!(userId || (postIds && postIds.length > 1));
```

Comment:
What's going on here?

Reply:
For the author analytics page it was still a bit to slow for my liking for people who have hundreds of posts, so I made it first fetch only cached (materialized) data and then do a followup fetch for the live data. For the post analytics page this isn't necessary because there is only one post so it's fast enough to begin with. The check here makes it do the two-step fetch only if there might be more than one post